src/layout/components/Page/index.vue

```diff
@@ -127,8 +127,9 @@ export default {
 
   .page-content {
     height: calc(100% - 20px);
-    overflow-x: hidden;
-    overflow-y: auto !important;
+    //Todo)) 注释掉的原因：当页面动态高度变化时就会导致页面最下方的按钮被 hidden 掉
+    //overflow-x: hidden;
+    //overflow-y: auto !important;
 
     ::v-deep > div {
       margin-bottom: 50px;
```

Comment:
The CSS code does not appear to have any issues or irregularities at this point. The main concerns would be regarding the visibility issue of the buttons on screen size changes. To fix this problem, consider using `position: absolute` inside `.page-content`, thus ensuring all elements are retained despite window size changes:

```css
@media (max-width: 768px) {
  .el-alert, ::v-deep > div {
    position: initial; /* Reset button positions after media query */
  }
}
```

Please replace `(min-width: 768px)` with whatever breakpoint works best for your project.

If there's anything specific you'd like help optimizing further, feel free to let me know!